ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs

```diff
@@ -48,8 +49,14 @@ public partial class UIManagerModule : ReactContextNativeModuleBase, ILifecycleE
             _window = window;
             _eventDispatcher = new EventDispatcher(reactContext);
             _uiImplementation = uiImplementation;
+            _uiImplementation.EventDispatcher = _eventDispatcher;
             _moduleConstants = CreateConstants(viewManagers);
             reactContext.AddLifecycleEventListener(this);
+
+            foreach (var eventEmitter in viewManagers.OfType<IEventEmitter>())
```

Comment:
Inject EventDispatcher into every ViewManager that implements the IEventEmitter interface... This is property injection, but it would be better to set this collaboration in the ViewManager's constructor...